### DISCUSSION TITLE
Theme

### DISCUSSION THREAD

**Discussion by Crepo920** [Godiesc/firefox-gx#222] [General]
Hii. I have question is there anyway that i can use here a black theme. i tried typing  Black but it remove the theme. if there is that would be so much help full.

![image](https://github.com/user-attachments/assets/570b451c-e4c4-4f22-8dc3-d1cc2362bb64)

**Comment by sasik007** [comment]
here you choose one of the colors available above


![Zrzut ekranu 2024-11-28 142050](https://github.com/user-attachments/assets/4f5c77f2-d395-4d83-a6dc-0b091d996120)

**Comment by Godiesc** [comment]
hi, that option is the color for the accent color, if your toolbar background color is black too so that color will be  invisible. for example you can use the '[gx-theme-style](https://addons.mozilla.org/es/firefox/addon/gx-theme-styles/)' extension without the needed of change the code:

PD: If you share a ScreenShot pointing the place you want to change would be perfect to help you.

![307611](https://github.com/user-attachments/assets/6d90c907-9af9-483d-bf3d-cc858541dcb6)

**Reply by Crepo920** [reply]
thnx
